app/tests/test_admin_page.py

```diff
@@ -723,3 +740,8 @@ def test_add_update_asset_invalid_media_type(self):
             Asset.objects.filter(item=self.item, name=sample["name"]).exists(),
             msg="Asset with invalid data has been added to db"
         )
+
+    @mock_s3_asset_file
+    def test_add_very_large_asset(self):
+        if settings.RUN_LARGE_ASSET_FILE_UNIT_TEST:
+            self._create_asset(self.item, large_file=True)
```

Comment:
Uses the unittest `@unittest.skipIf(settings.RUN_LARGE_ASSET_FILE_UNIT_TEST == False)` instead here, this way you also have a proper test output. You could maybe then also change the setting like this `SKIP_LARGE_ASSET_FILE_UNITTEST` ;-)

Reply:
Now this whole problem is fixed in a much shorter way